export default function ErrorThrow() {
  throw new Error("ErrorThrow");
  return <p>正しく実行されました</p>;
}
